### PR TITLE
add more craftengings recipes to missions

### PR DESCRIPTION
**Pull request by funkychicken493**

### CODE REVIEW

**Comment by funkychicken493** [comment]
oops i mispelde carftienger ereicpes

**Comment by Ph4i1ur3** [comment]
Can we crank these values up a bit? Ideally 3 tiers: Easy, Hard, and KYS

**Comment by funkychicken493** [comment]
I can organize that

**Comment by funkychicken493** [comment]
I put this in draft mode for now so i can work on it more

**Comment by funkychicken493** [comment]
@Ph4i1ur3 review time

File: denizen_scripts/HeroCraft/missions/PvE/mission_craft.dsc [hunk 1]
```diff
@@ -13,21 +13,79 @@ mission_craft:
     max: mission_craft_complete
   rewards:
     daily: 100
-    weekly: 200
-    monthly: 400
+    weekly: 400
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/missions/PvE/mission_craft.dsc]: Rewards are too high, weekly is multiplied by 7, and monthly by 30

Comment (funkychicken493) [reply denizen_scripts/HeroCraft/missions/PvE/mission_craft.dsc]: oh.

**Pull request merged**